docs/_interface/hardware.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Hardware requirements
+sort-key: M
+---
+
+The hardware requirements for running an Ethereum node depend upon the node configuration and can change over time as upgrades to the network are implemented. Ethereum nodes can be run on low power, resource-constrained devices such as Raspberry Pi's. Prebuilt, dedicated staking machines are available from several companies - these might be good choices for users who want plug-and-play hardware specifically designed for Ethereum. However, many users will choose to run nodes on laptop or desktop computers. Those users need to know the basic minimum requirements.
```

Comment:
`running an Ethereum node` -> `running a geth-node`, IMO. We're geth, we're talking about geth requirements, not erigon or besu.

Reply:
yep, thanks

Reply:
```suggestion
The hardware requirements for running a Geth node depend upon the node configuration and can change over time as upgrades to the network are implemented. Ethereum nodes can be run on low power, resource-constrained devices such as Raspberry Pi's. Prebuilt, dedicated staking machines are available from several companies - these might be good choices for users who want plug-and-play hardware specifically designed for Ethereum. However, many users will choose to run nodes on laptop or desktop computers. Those users need to know the basic minimum requirements.
```